src/main/java/org/checkerframework/checker/dependencyinjection/DependencyInjectionVisitor.java

```diff
@@ -13,4 +20,31 @@ public DependencyInjectionVisitor(BaseTypeChecker c) {
   protected DependencyInjectionAnnotatedTypeFactory createTypeFactory() {
     return new DependencyInjectionAnnotatedTypeFactory(this.checker);
   }
+
+  @Override
+  public Void visitMethod(MethodTree tree, Void p) {
+    ExecutableElement element = TreeUtils.elementFromDeclaration(tree);
+    AnnotationMirror annotation =
+        this.getTypeFactory().getDeclAnnotation(element, com.google.inject.Inject.class);
+
+    if (annotation != null) {
+      DependencyInjectionAnnotatedTypeFactory.addDependency(annotation.toString(), element);
```

Comment:
This doesn't look correct (the ATF's method is called `addInjectionPoint`). Please make sure that the code compiles before asking for a review.

Reply:
I actually did make this change, I didn't realize I didn't include it in the commit. Should be good now.